### DISCUSSION TITLE
Range bar chart

### DISCUSSION THREAD

**Discussion by Carbonblack8** [tradingview/lightweight-charts#1050] [Q&A] [answered]
TradingView has range bars  (instead of candlesticks, for example) as an option. Does Lightweight? I haven’t been able to find it

**Comment by romfrancois** [comment, marked as answer]
Hi @Carbonblack8 !

Unfortunately not. All available types of series can be found [here](https://tradingview.github.io/lightweight-charts/docs/next/series-types).

**Reply by Carbonblack8** [reply]
Thank you!
